### DISCUSSION TITLE
Page "is having problems"

### DISCUSSION THREAD

**Discussion by ChrisK91** [community/community#21823] [Other Feature Feedback, Questions, & Ideas]
<p>I have a Jekyll page located in a private repository, that has been working fine. However, when I pushed a new article today, I received the message “Your site is having problems building: Page build failed. For more information, see <a href="https://help.github.com/en/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#troubleshooting-build-errors" rel="nofollow noopener">https://help.github.com/en/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#troubleshooting-build-errors</a>.”</p>
<p>The repository is private, I have a Pro account.</p>
<p>The repository size is around 27 MB.</p>
<p>The site runs fine with even after <em>bundle update</em> and <em>bundle exec jekyll serve</em>, and displays no errors when built locally. Is there a way to get a more detailed error log?</p>
<p>I have a reference to jekyll-paginate in the config, however that is listed under <a href="https://pages.github.com/versions/" rel="nofollow noopener">https://pages.github.com/versions/</a></p>

**Comment by ChrisK91** [comment, marked as answer]
<p>I have managed to resolve the error. I had a link to an uncomitted file, which was present locally. This caused an error during build. It would however been helpful, to access the build log while debugging this.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/chrisk91">@chrisk91</a>, Thanks for being here and coming back top share what worked for you!</p>
